src/strategies/jelly-ve-nft-balance/index.ts

```diff
@@ -0,0 +1,56 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+
+export const author = 'JellyProtocol';
+export const version = '0.1.3';
```

Comment:
```suggestion
export const version = '0.1.0';
```

Reply:
I'm going to make this strategy more complicated once I finish implementing our veNFTs. I wanted to have this strategy so I can test